src/Components/Facility/InventoryLog.tsx

```diff
@@ -146,8 +146,8 @@ export default function InventoryLog(props: any) {
           </p>
         </td>
         <td>
-          <Tooltip
-            title={
+          <ToolTip
```

Comment:
The custom tooltip component also is deprecated if you've read the doc comments of that component. Refer #5006 on how to replace each component properly.

For tooltip we are using classes over component because, less JSX the better!